### DISCUSSION TITLE
Allow only certain files in File Picker

### DISCUSSION THREAD

**Discussion by edryansaltat** [chatwoot/chatwoot#7661] [Feature Requests]
We’re using Chatwoot that sometimes collect pdf or images from our chat users. We would like to allow those certain file types only, to avoid any potential security issues.

**Comment by jtayrone** [comment]
This is a type of custom functionality. If you wish, you'll have to do it directly in your version and modify it to meet your needs. I suggest searching the community to see if anyone has done something similar.
